src/modules/notifications/components/assets/info-lg.svg

```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bi bi-info-lg" viewBox="0 0 16 16">
```

Comment:
@alsturm  License is missing

Reply:
@alsturm I guess this asset is obsolete, isn't? So we can remove it.

Reply:
remove it.